siteconfig.json5

```diff
@@ -3,6 +3,9 @@
     // `true` if the button should be hidden and `false` otherwise.
     hideQssSaveButton: false,
 
+    // File containing default values which are used after resetting user settings
+    defaultValues: "C:\\Users\\vagrant\\AppData\\Roaming\\gpii\\defaultSettings.json5",
```

Comment:
I'd say we need to use the %appdata% env variable to come up with the real path to the file.

Reply:
Done :-)